jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-backend/src/main/java/org/jbpm/workbench/cm/backend/server/RemoteCaseManagementServiceImpl.java

```diff
@@ -46,10 +55,14 @@
 public class RemoteCaseManagementServiceImpl implements CaseManagementService {
 
     public static final int PAGE_SIZE_UNLIMITED = Integer.MAX_VALUE;
+    public static final String[] NODE_TYPE_HUMAN_TASK = new String[]{"Human Task", "HumanTaskNode"};
```

Comment:
Instead of creating an array, you could create the list directly with ` Arrays.asList("Human Task", "HumanTaskNode")`. That way you can avoid creating the list later on.